src/sgame/sg_utils.cpp

```diff
@@ -945,3 +945,31 @@ glm::vec3 G_CalcMuzzlePoint( const gentity_t *self, const glm::vec3 &forward )
 	// Meh. I doubt it saves much. Casting to short ints might have, though. (copypaste)
 	return glm::floor( muzzlePoint + 0.5f );
 }
+
+//Dunno which ones are really useful, CBSE is not even self-contained.
+#include "sg_local.h"
+#include "Entities.h"
+#include "CBSE.h"
+#include "sg_cm_world.h"
+
+bool gentity_t::Damage( float amount, gentity_t* source,
+		Util::optional<glm::vec3> location,
+		Util::optional<glm::vec3> direction,
+		int flags, meansOfDeath_t meansOfDeath )
+{
+	if ( s.eType != entityType_t::ET_MOVER )
+	{
+		return entity->Damage( amount, source, location, direction, flags, meansOfDeath );
+	}
+
+	if ( pain )
+	{
+		pain( this, source, amount );
+	}
+	health -= amount;
+	if ( health <= 0 && die )
+	{
+		die( this, nullptr, source, 0 );
+	}
+	return true; // seriously, not sure what's this about
```

Comment:
When you send a message a 'true' return means the entity had a handler for that message

Reply:
So, should return false? What are the implications, exactly?
IIRC, the return value is used exactly one time, for rocketpods. This is almost like a "return ent == rocketpod".
But do you prefer me to return false here?
There is no documentation around, so i basically tried to mimic what appeared to be previous behavior (should be explicit with that comment) , as I wanted this 8 years old regression fixed. Originally I didn't really intended to upstream it though.

Reply:
I suppose it should return true iff the mover has health.

Reply:
Sounds reasonable. I'll return true if the mover have "config->health != 0", that is, maxHealth.